### DISCUSSION TITLE
Install failed on Windows 10 + strawberry-perl-5.32.0.1-64bit

### DISCUSSION THREAD

**Discussion by yarnik1** [ImageMagick/ImageMagick#2769] [Development]
When I do cmd and # cpan Image::Magick:
```
....
Checking if your kit is complete...
Looks good
Generating a gmake-style Makefile
Writing Makefile for Image::Magick
Writing MYMETA.yml and MYMETA.json
  JCRISTY/PerlMagick-6.89-1.tar.gz
  C:\Strawberry\perl\bin\perl.exe Makefile.PL -- OK
Running make for J/JC/JCRISTY/PerlMagick-6.89-1.tar.gz
CPAN: Module::CoreList loaded ok (v5.20200717)
cp Magick.pm blib\lib\Image\Magick.pm
AutoSplitting blib\lib\Image\Magick.pm (blib\lib\auto\Image\Magick)
Running Mkbootstrap for Magick ()
"C:\Strawberry\perl\bin\perl.exe" -MExtUtils::Command -e chmod -- 644
"Magick.bs"
"C:\Strawberry\perl\bin\perl.exe" -MExtUtils::Command::MM -e cp_nonempty --
Magick.bs blib\arch\auto\Image\Magick\Magick.bs 644
"C:\Strawberry\perl\bin\perl.exe" "C:\Strawberry\perl\lib\ExtUtils/xsubpp"
 -typemap C:\STRAWB~1\perl\lib\ExtUtils\typemap -typemap
C:\STRAWB~1\cpan\build\PerlMagick-6.89-2\typemap  Magick.xs > Magick.xsc
"C:\Strawberry\perl\bin\perl.exe" -MExtUtils::Command -e mv -- Magick.xsc
Magick.c
gcc -c   -DWIN32 -DWIN64 -D__USE_MINGW_ANSI_STDIO -DPERL_TEXTMODE_SCRIPTS
-DPERL_IMPLICIT_CONTEXT -DPERL_IMPLICIT_SYS -DUSE_PERLIO -fwrapv
-fno-strict-aliasing -mms-bitfields -s -O2   -DVERSION=\"6.89\"
-DXS_VERSION=\"6.89\"  "-IC:\STRAWB~1\perl\lib\CORE"  -D_LARGE_FILES=1
-DHAVE_CONFIG_H Magick.c
Magick.xs:60:10: fatal error: magick/MagickCore.h: No such file or directory
 #include <magick/MagickCore.h>
          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
gmake: *** [Makefile:347: Magick.o] Error 1
  JCRISTY/PerlMagick-6.89-1.tar.gz
  C:\STRAWB~1\c\bin\gmake.exe -- NOT OK
Stopping: 'install' failed for 'Image::Magick'.
```

**Comment by urban-warrior** [comment]
Grab the Windows release of ImageMagick from https://imagemagick.org/download/windows.  It includes a release of PerlMagick that will compile and run against the ImageMagick MagickCore API.

**Comment by 1nelly1** [comment]
I have the same problem. Not succeeding installing ImageMagick in Strawberry Perl. 5.32.1 32-bit on 64-bit Windows.
Can please somebody explain. What are the main steps? I tried also with ImageMagick-7.0.11-5-Q16-HDRI-x64-dll.exe and ImageMagick-7.0.11-5-Q16-HDRI-x86-dll.exe. I also tried with ppm install Image::Magick but then there is always the message that a PPD file can be read but is not intended for my Perl version. Where can I see for what Perl version the PPD file is intended? Where is the ppd-file?
Thank you for helping

**Reply by twata1** [reply]
Regarding Strawberry Perl 5.30.2 64bit, it may be useful.

When I install ImageMagick-7.0.11-10-Q16-HDRI-x64-dll.exe, I check the following:

[x] Install legacy utilities (e.g. convert)
[x] Install development headers and libraries for C and C++

And I install Visual C++ Redistributable Package if needed.

And I download Image-Magick-7.0.11.tar.gz from metacpan,
and extract Image-Magick-7.0.11.tar.gz, and apply the attached patch or edit the Makefile.PL directly.

And
```
perl Makefile.PL
gmake
gmake test
gmake install
```

I hope this will help.

[Image-Magick-7.0.11-Strawberry-Perl.patch.txt](https://github.com/ImageMagick/ImageMagick/files/6410767/Image-Magick-7.0.11-Strawberry-Perl.patch.txt)
